contrib/qos/README.md

```diff
@@ -1,5 +1,5 @@
 ### QoS (Quality of service) ###
 
-This is a Linux bash script that will set up tc to limit the outgoing bandwidth for connections to the vBitcoin network. It limits outbound TCP traffic with a source or destination port of 18333, but not if the destination IP is within a LAN.
+This is a Linux bash script that will set up tc to limit the outgoing bandwidth for connections to the vBitcoin network. It limits outbound TCP traffic with a source or destination port of 8333, but not if the destination IP is within a LAN.
```

Comment:
should we not be using 18333 ? We don't want the port to conflict with bitcoin no?
I think bitcoin is 8332 and 8333